bin/configure.sh

```diff
@@ -12,10 +12,26 @@
 # - ZWE_zowe_runtimeDirectory
 # - ZWE_zowe_workspaceDirectory
 
-cd ${ZWE_zowe_runtimeDirectory}/components/app-server/share/zlux-app-server/bin
-. ./convert-env.sh
+COMPONENT_HOME=${ZWE_zowe_runtimeDirectory}/components/zss
 
-cd ${ZWE_zowe_runtimeDirectory}/components/app-server/share/zlux-app-server/lib
+# this is to resolve (builtin) plugins that use ZLUX_ROOT_DIR as a relative path. if it doesnt exist, the plugins shouldn't either, so no problem
+if [ -z "${ZLUX_ROOT_DIR}" ]; then
+  if [ -d "${COMPONENT_HOME}/../app-server/share" ]; then
+    export ZLUX_ROOT_DIR=$(cd `dirname ${COMPONENT_HOME}/../app-server/share/zlux-app-server` && pwd)
+  fi
+fi
+
+# Consume server infrastructure environment variables if possible
+if [ -n "$ZLUX_ROOT_DIR" ]; then
+  cd ${ZLUX_ROOT_DIR}/zlux-app-server/bin/init
+  . ../utils/convert-env.sh
+  NO_NODE=1
+  CHECK_PC_BIT=1
+  . ./plugins-init.sh $NO_NODE $CHECK_PC_BIT
```

Comment:
Add a code comment about how this is being called in one case and not the other because the other is a TODO for future packaging options that i'm unsure about... maybe just comment the whole other case out?